recipes/scikit-allel/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "scikit-allel" %}
+{% set version = "0.21.2" %}
+{% set sha256 = "589ca19ca5bc083fe3a9318e5e2d9d415f9694fbea8293e901e756aecbaef9b6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
+  run:
+    - python
+    - numpy
```

Comment:
Sorry sorry, just read http://conda.pydata.org/docs/building/meta-yaml.html#run and see you literally mean "numpy x.x", will change...